### PR TITLE
Cypress/E2E: Fix slash commands spec and some promotions

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
 - Fixed slash commands spec
 - Some promotions

#### Ticket Link
NA

### CODE REVIEW

File: e2e/cypress/integration/enterprise/system_console/compliance/download_compliance_export_file_spec.js [hunk 1]
```diff
@@ -7,6 +7,7 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
+// Stage: @prod
```
Comment (furqanmlk) [inline e2e/cypress/integration/enterprise/system_console/compliance/download_compliance_export_file_spec.js:10]: Hi @josephbaylon 

I have open PR for compliance export test cases https://github.com/mattermost/mattermost-webapp/pull/7503
you want me to create new files for new test cases ? cos I added some new test cases in this file as it was not marked as prod.
Please advise

Comment (furqanmlk) [inline e2e/cypress/integration/enterprise/system_console/compliance/download_compliance_export_file_spec.js:10]:
```suggestion
```

**Review by furqanmlk — APPROVED** [review]
Thank @josephbaylon 
All looks good :)

**Review by prapti — APPROVED** [review]
👍

**Pull request merged**